src/admin-views/tribe-options-tickets.php

```diff
@@ -272,8 +272,9 @@
 	'ticket-paypal-success-page'      => array(
 		'type'            => 'dropdown',
 		'label'           => esc_html__( 'Success page', 'event-tickets' ),
-		'tooltip'         => esc_html__( "After a successful PayPal order users will be redirected to this page; use the [{$tpp_success_shortcode}] shortcode to display the order confirmation to the user in the page content.",
-			'event-tickets' ),
+		'tooltip'         => sprintf(
```

Comment:
Thanks for this fix @andrasguseo 

Just a small thing regarding the formatting of your params in the function to make it more easier to spot what params are used. 

Would you mind update with this format instead.

```php
sprintf(
	esc_html__( 
		"text",
		'event-tickets'
	)
	'[$tpp_success_shortcode]'
)
```

Thank you

Reply:
Thanks! I'm always struggling with the tabs and spaces. :)

Reply:
Great let me know if I can help with anything but you might want to type it as if it might be wrong from GH so is easier if you use your tabs on your editor to update accordingly.

Reply:
Can you please check now @mitogh ?